specs/www.rst

```diff
@@ -149,6 +149,29 @@ Keys:
   will be ignored. Optional, defaults to ``False``.
 
 
+Server Push
+'''''''''''
+
+Indicates that the server should send a server push (or push promise)
+to the client. This message can be sent at any time after the
+*Response Start* message but before the final *Response Body* message.
```

Comment:
So, Server Push responses need a body if I recall correctly; are you proposing this to be the type of push where the application sends back what it wants pushed and then the server calls itself with the given path and headers to get the content?

Reply:
Yep, I've updated to hopefully make this clear.